os-downstream-tag.py

```diff
@@ -114,7 +114,7 @@ def main():
         "{}/stable/{}".format(upstream_remote, release),
         "{}/unmaintained/{}".format(upstream_remote, release),
         "{}-eol".format(release),
-        str(release),
+        "{}/{}".format(upstream_remote, release),
```

Comment:
Based on example failure, I think we also need to add upstream_remote prefix to eol case.
Not significant in our current use case, but maybe for some other works in the future.

Reply:
Ah that's right. I forgot they are tags rather than branches

Reply:
@seunghun1ee see https://github.com/stackhpc/naming-things-is-hard/pull/13 for the eol case